_posts/2023-06-20-trino-fest-2023-recap.md

```diff
@@ -0,0 +1,122 @@
+---
+layout: post
+title: "Trino Fest 2023 recap"
+author: "Manfred Moser, Cole Bowden"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2023/trino-fest.png
+---
+
+Last week we held Trino Fest, and we did not find time to chill at the
+lakehouse. Despite the theme of a summercamp for lakehouse analytics, the fest
+kept us all busy. Great demos, amazing announcements, new plugins, and use cases
+reached our active audience. Thanks go to our event host and organizer
+[Starburst](https://www.starburst.io/), our sponsors
+[AWS](https://aws.amazon.com/) and [Alluxio](https://www.alluxio.io/), and many
+well-prepared speakers, and our great live audience. Now you get a chance to
+catch up on anything you missed.
+
+<!--more-->
+
+<div class="container">
+  <div class="row">
+    <div class="col-sm">
+      <a href="https://www.starburst.io/">
+        <img src="{{site.url}}/assets/images/logos/starburst-small.png" title=" Starburst, event host and organizer ">
+      </a>
+    </div>
+    <div class="col-sm">
+      <a href="https://www.alluxio.io/">
+        <img src="{{site.url}}/assets/images/logos/alluxio-small.png" title="Alluxio, event sponsor">
+      </a>
+    </div>
+    <div class="col-sm">
+      <a href="https://aws.amazon.com/">
+        <img src="{{site.url}}/assets/images/logos/aws-small.png" title="AWS, event sponsor">
+      </a>
+    </div>
+  </div>
+</div>
+
+In the weeks running up to the event we had numerous blog posts going out, and
+racked up great interest in the Trino community and beyond. Over 1100
+registrations blew away our numbers from last year. More importantly, during the
+two half days of the event we had up to over 560 attendees watching live and
```

Comment:
```suggestion
two half-days of the event, we had over 560 attendees watching live and
```

Reply:
we can do "up to" or "over" but not both 😛

Reply:
hahaha